solutions/tc-submultiplesofn.mdx

```diff
@@ -69,47 +69,47 @@ class SubmultiplesOfN {
 import java.util.Arrays;
 
 public final class SubmultiplesOfN {
-    private static final int MOD = (int) 1e9 + 7;
-    public static int count(String strNum, int div) {
-        int[] num = new int[strNum.length()];
-        for (int i = 0; i < strNum.length(); i++) {
-            num[i] = Character.getNumericValue(strNum.charAt(i));
-            if (num[i] < 0 || num[i] > 9) {
-                throw new IllegalArgumentException("invalid number given for argument num");
-            }
-        }
+	private static final int MOD = (int) 1e9 + 7;
+	public static int count(String strNum, int div) {
+		int[] num = new int[strNum.length()];
+		for (int i = 0; i < strNum.length(); i++) {
+			num[i] = Character.getNumericValue(strNum.charAt(i));
+			if (num[i] < 0 || num[i] > 9) {
+				throw new IllegalArgumentException("invalid number given for argument num");
```

Comment:
Is this necessary? I've never seen people throw exceptions in cp code, though I don't think it matter that much.

Reply:
nothing wrong with a little input validation